go-app-ussd_sassa_registration.js

```diff
@@ -0,0 +1,203 @@
+var go = {};
+go;
+
+go.app = (function () {
+  var _ = require("lodash");
+  var moment = require("moment");
+  var vumigo = require("vumigo_v02");
+  var App = vumigo.App;
+  var Choice = vumigo.states.Choice;
+  var EndState = vumigo.states.EndState;
+  var MenuState = vumigo.states.MenuState;
+  var FreeText = vumigo.states.FreeText;
+  var ChoiceState = vumigo.states.ChoiceState;
+  var JsonApi = vumigo.http.api.JsonApi;
+
+  var GoNDOH = App.extend(function (self) {
```

Comment:
`GoSASSA` perhaps?

Reply:
The init file expects it to be NDOH, this will change when we move it out some day